mantidproject/mantid
# Enable the graphical selection tool in the release build

**Do not merge before #34619 and ~~#34326~~**

**Report to:** Max and Becky at ISIS

**Description of work.**
This PR:
- exposes the new Reduction Preview tab in the reflectometry GUI in the release build
- adds documentation and release notes for the new tab

Fixes #34577

**To test:**

- Do a Release build (to check the new tab is now visible)
- Follow the test instructions on #31069
- Ensure the new documentation reads ok and makes sense with the workflow (note that it is deliberately minimal at the moment because the tab is still under development).

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
